ecc/enclave/enclave_stub.go

```diff
@@ -28,7 +29,8 @@ import (
 
 	"github.com/golang/protobuf/proto"
 	"github.com/hyperledger-labs/fabric-private-chaincode/ecc/crypto"
-	"github.com/hyperledger-labs/fabric-private-chaincode/ecc/tlcc"
+	//"github.com/hyperledger-labs/fabric-private-chaincode/ecc/tlcc"
```

Comment:
Let's just purge it instead of commenting out given that we have purged this code from master of repo?
```suggestion
```